src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/JSON1001.SyntaxError.sarif

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.2.json",
   "version": "2.1.0",
```

Comment:
>toolExecutionNotifications [](start = 11, length = 26)

Here's an example of the 'analyzing file' verbose message that had snuck into our log files, now removed.